Makefile.prog

```diff
@@ -28,4 +28,3 @@ clean:
 
 ${PROG}:${SRCS:.c=.o}
 	${CC} -o ${PROG} ${SRCS:.c=.o} ${LDFLAGS} ${LDADD_EXTRA} ${LDADD_REQ} ${LDADD_POSIX}
-
```

Comment:
Oops, this will change all the generated `Makefile`s.  I'm working on it.

Reply:
Fixed, but I'll take down the PR because I'm likely to find more style things to add.